src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts

```diff
@@ -168,8 +166,9 @@ describe('ContainerSelectorComponent', () => {
         const paginatorService: PaginatorService = de.injector.get(PaginatorService);
         spyOn(paginatorService, 'getWithOffset').and.returnValue(observableOf(containers));
         comp.handleFilterChange('');
-
-        expect(comp.currentContainers[0].identifier).toEqual('427c47a4-c380-439f');
-        expect(comp.currentContainers[1].identifier).toEqual('container/path');
+        comp.currentContainers.subscribe((items: DotContainer[]) => {
+            expect(items[0].identifier).toEqual('427c47a4-c380-439f');
+            expect(items[1].identifier).toEqual('container/path');
```

Comment:
I think you should do this againts the HTML attr.

Reply:
done